### ISSUE TITLE
Pharo build is broken

### ISSUE THREAD

**Issue by Ducasse** [guillep/pharo-core#17332]
Apparently the build is broken.
Now I do not understand why because I did not change anything in the pharo13 branch of Microdown. 

```
NewUndeclaredWarning: Microdown class>>asRichText: (MicRichTextComposer is Undeclared)
NewUndeclaredWarning: MicElement>>propertyAt: (MicPropertyError is Undeclared)
NewUndeclaredWarning: MicElement>>removeProperty: (MicPropertyError is Undeclared)
NewUndeclaredWarning: MicFigureBlock>>hasCaption (captionElements is Undeclared)
MetacelloNotification: Loaded -> Microdown --- github://pillar-markup/Microdown:Pharo13/src(e390aeb)
MetacelloNotification: Loaded -> Microdown-Tests --- github://pillar-markup/Microdown:Pharo13/src(e390aeb)
MetacelloNotification: Loaded -> Microdown-RichTextComposer --- github://pillar-markup/Microdown:Pharo13/src(e390aeb)
 postload [31mUndeclaredVariableRead: Attempt to read undeclared variable IceRepository
[0mUndeclaredVariable>>runtimeUndeclaredReadInContext:
BaselineOfMicrodown>>postload:package:
[ :aLoader :pkgSpec | self project configuration perform: selector with: aLoader with: pkgSpec ] in MetacelloVersionSpec(MetacelloSpec)>>doItBlock: in Block: [ :aLoader :pkgSpec | self project configurati[..]
FullBlockClosure(BlockClosure)>>valueWithPossibleArgs:
MetacelloLoadTarget>>visitPostLoadDirective:
MetacelloPostLoadDirective>>acceptVisitor:
[ :dir |
		dir acceptVisitor: self ] in MetacelloLoadTarget>>visitLinearLoadDirective: in Block: [ :dir |...
OrderedCollection>>do:
MetacelloLoadTarget>>visitLinearLoadDirective:
MetacelloLinearLoadDirective>>acceptVisitor:
MetacelloVersion>>loadRequiredFromArray:withEngine:
[
```

**Comment by Ducasse** [comment]
So apparently this is not in the microdown baseline in Pharo but in the baseline in the microdown repo.

**Comment by Ducasse** [comment]
I do not get why  Attempt to read undeclared variable IceRepository

```
BaselineOfMicrodown >> postload: loader package: packageSpec

	| p |
	self class name, ' postload ' traceCr. 

	p := (IceRepository repositoryNamed: 'microdown').
	p ifNil: [ p :=  (IceRepository repositoryNamed: 'Microdown') ].
	[ 
	(p ifNotNil: [ :p2 | (p2 packageNamed: 'BaselineOfMicrodown') reload ]) 
	] 
		on: MCMergeOrLoadWarning 
		do: [ :ex | ex load ]
```

**Comment by hernanmd** [comment]
Something like this?

```smalltalk
postload: loader package: packageSpec

	| repo |
	self class name, ' postload ' traceCr. 
	repo := (self class environment classNamed: #IceRepository)
		        ifNotNil: [ :iceberg |
			        iceberg registry
				        detect: [ :each | each name = 'Microdown' ]
				        ifNone: [ nil ] ].

	[ repo ifNotNil: [ :p2 | (p2 packageNamed: 'BaselineOfMicrodown') reload ] ] 
	on: MCMergeOrLoadWarning 
	do: [ :ex | ex load ]
```

**Comment by jecisc** [comment]
I did a PR to fix this: https://github.com/pillar-markup/Microdown/pull/909

**Comment by hernanmd** [comment]
Fixed, although some problems persists: #17348
